packages/venia-ui/lib/components/Header/accountTrigger.js

```diff
@@ -0,0 +1,53 @@
+import React, { Fragment } from 'react';
+import { shape, string } from 'prop-types';
+
+import { useAccountTrigger } from '@magento/peregrine/lib/talons/Header/useAccountTrigger';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import SIGN_OUT_MUTATION from '../../queries/signOut.graphql';
+import AccountChip from '../AccountChip';
+import AccountMenu from '../AccountMenu';
+import defaultClasses from './accountTrigger.css';
+
+const AccountTrigger = props => {
+    const {
```

Comment:
Can you separate out object destruction into `talonProps`?

Reply:
Resolved by https://github.com/magento/pwa-studio/commit/767ec002ca748aa6e7467f79a5427c84950b4c00.